content/realtime/channels.textile

```diff
@@ -236,6 +236,147 @@ channel.subscribe { message in
 }
 ```
 
+h4(#rewind). Subscribing to a channel and fetching history with rewind
+
+If you wish to both subscribe to a channel, as well as obtain some messages from said channel's "history":/realtime/history seamlessly, you can make use of the "rewind":/realtime/channel-params feature. This will result in the specified number of messages from "history":/realtime/history being returned as part of the subscription process.
+
+minimize. View details
```

Comment:
I think it's ok to keep in maximized. What are your thoughts?

Reply:
My thinking was that as channel is a very introductory page, it'd been good to avoid pushing bits such as `publishing` too far down the page. Happy to go either way though, do you think the size is an issue?

Reply:
Fair point, let's leave the minimize then.